testing/flakybot-function/main.py

```diff
@@ -75,26 +73,55 @@ def get_elapsed_time(step):
 
 
 def send_to_flakybot(event, context):
-    try:
-        data = base64.b64decode(event['data'])
-        build_event = json.loads(data.decode("utf-8").strip())
+    # Allow function framework to do all needed handling.
+    data = base64.b64decode(event['data'])
+    build_event = json.loads(data.decode("utf-8").strip())
```

Comment:
Should we keep this in the try statement  for the off chance the Pub/Sub message?

Reply:
I moved this out because it needs separate handling from the stuff below, because we don't have the build metadata yet. Then I realized: the functions framework should handle generic exceptions correctly so may as well not include it in a try at all.

Reply:
Hrm.  If you put it in the `try` statement, it'll be included in the structured error logging.